src/misc.rs

```diff
@@ -6,7 +6,7 @@ use crate::attribute::{Attribute, CkAttrs};
 use crate::error::Result;
 use crate::interface::*;
 use crate::object::{Object, ObjectFactories, ObjectType};
-
+use crate::ossl::common::zeromem as ossl_zeromem;
```

Comment:
This pulls openssl into the common, which might be an issue if we would like to support some different backend. Likely not an issue now, but more like a question/though, if this could be done in some crypto agnostic manner (memset?).